app/controllers/bz_controller.rb

```diff
@@ -673,18 +673,24 @@ def user_retained_data_batch
 
     data = {}
     if params[:names]
```

Comment:
Whoops, these changes were actually the stuff from this commit: [Optimize loading a page or assignment retained data (aka magic field).](https://github.com/beyond-z/canvas-lms/commit/63ca41805cd19d596248de6ce5259370d03d7d51)